src/messenger.rs

```diff
@@ -89,20 +87,29 @@ impl Messenger for SlackMessenger {
 
 impl SlackMessenger {
     fn send_to_slack(&self, channel: &str, msg: &str, attachments: &Vec<SlackAttachment>) {
-        // user desires peace and quiet. do not disturb!
-        if channel == DND_MARKER || channel == users::mention(DND_MARKER) {
-            return;
-        }
-
         if let Err(e) = self.slack.send(slack::req(channel, msg, attachments.clone())) {
             error!("Error sending to slack worker: {}", e);
         }
     }
 
     fn send_to_slackbots(&self, users: Vec<github::User>, msg: &str, attachments: &Vec<SlackAttachment>) {
         for user in users {
-            let slack_ref = self.config.users().slack_user_ref(user.login());
-            self.send_to_slack(slack_ref.as_str(), msg, attachments);
+            let slack_ref = match self.config.users().lookup_info(user.login()) {
```

Comment:
ugh. this is kinda ugly. Would be much eaiser to clean this up if we decided to do #160...